fastplotlib/graphics/_base.py

```diff
@@ -413,3 +422,21 @@ def rotate(self, alpha: float, axis: Literal["x", "y", "z"] = "y"):
                 f"`axis` must be either `x`, `y`, or `z`. `{axis}` provided instead!"
             )
         self.rotation = la.quat_mul(rot, self.rotation)
+
+    @property
+    def axes(self) -> Axes:
+        return self._axes
+
+    def add_axes(self):
```

Comment:
for images, adding axes places them on the top of the image with the numbers above the image but for the y-axis the numbers are overlayed on top of the image...I think the number should also be to the outside of the image here

![image](https://github.com/user-attachments/assets/b975d5f0-b56b-483d-a3c7-2eae16dfb614)

Reply:
if I flip the image, the axes are correct...is there any easy way to make sure no matter the orientation of the image we are also displaying axes for images on the bottom of the image and on the left??

Reply:
A user can call `graphic.axes.update_using_camera()` after flipping. We can't do this automatically without making it an animation function or having callbacks from cameras. Let's wait for https://github.com/pygfx/pygfx/issues/495 before persuing this further.